tests/test_oidcop.py

```diff
@@ -545,7 +545,8 @@ def test_handle_authn_response_authcode_flow(self, context, frontend, authn_req)
         context.request = {
             'grant_type': 'authorization_code',
             'redirect_uri': CLIENT_RED_URL,
-            'client_id': CLIENT_AUTHN_REQUEST['client_id'],
```

Comment:
I would keep the original test + add a new (regression) test without `client_id`, to make sure that both cases work.

Reply:
Sure, done.